webapp/src/routes/Home/ShowLifebanks.js

```diff
@@ -197,18 +197,15 @@ const ShowLifebanks = ({ banks, loading, isDesktop }) => {
     <LinkRouter
       style={{ textDecoration: 'none' }}
       to={{
-        pathname: `info/${props.bank.user.username.replaceAll(' ', '-')}`,
+        pathname: `info/${props.bank.userName.replaceAll(' ', '-')}`,
         state: { profile: props.bank, isLifebank: true }
       }}
     >
       <ListItem className={classes.listItem} button>
         <ListItemAvatar>
           <Avatar
-            src={
-              props.bank.info.logo_url !== ''
-                ? `//images.weserv.nl?url=${props.bank.info.logo_url}&h=60&dpr=1`
-                : ''
-            }
+            src={`//images.weserv.nl?url=${props.bank.logo || ''
```

Comment:
In this case, it is better to maintain the validation to avoid something like //images.weserv.nl?url=''&h=60&dpr=1

Reply:
So, if the image is null or '', we wouldn't need to render or use src